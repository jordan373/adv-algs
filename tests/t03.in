7
addEdge 0 2 5
addEdge 0 1 3
addEdge 1 3 6
addEdge 2 3 2
addEdge 2 4 7
addEdge 2 5 3
addEdge 2 6 7
addEdge 3 5 9
addEdge 4 5 5
addEdge 4 6 2
addEdge 5 6 1
printGraph
dijkstra 0
printDistance
printParents
printPath 0 4
lenghtShortestW 0 4
