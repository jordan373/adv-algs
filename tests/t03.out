Graph is:
2 1 
0 3 
0 3 4 5 6 
1 2 5 
2 5 6 
2 3 4 6 
2 4 5 
Running the Dijkstra's algorithm on 0
Distances:
0 3 5 7 11 8 9 
Parents:
0 0 0 2 6 2 5 
Path from 0 to 4 is:
0 2 5 6 4 
Calculating length of the shortest path from 0 to 4
0 2 5 6 4 
The length is 11
